backend/mora/util.py

```diff
@@ -283,19 +285,26 @@ def update_config(mapping, config_path, allow_environment=True):
             mapping[key] = overrides[key]
 
     except IOError:
-        print('Unable to read config {}'.format(config_path))
+        logger.error('Unable to read config {}'.format(config_path))
 
     if allow_environment:
         overrides = {
-            k[5:]: v
+            k[6:]: v
             for k, v in os.environ.items()
             if k.startswith('OS2MO_')
         }
 
         for key in overrides.keys():
-            print(
-                ' * Using override OS2MO_{}={!r}'.format(key, overrides[key]),
-                file=sys.stderr)
+            if overrides[key] in ['True', 'true']:
```

Comment:
Utility function? Also, we have another function which does _almost_ the same; `get_args_flag()` — perhaps they should share the logic?

(`TRUE`, `1`, and `y` are common ways to opt into something. `fALsE` is just being obnoxious 😉)

Reply:
I agree somewhat, but I also think what i did here was maybe wrong.:
A config file is different - it may relay settings to other modules that actually need 1, 'true' or y. 
-
Hence I may even back off from this and move it into 'check_config', so it is the checker, that does the conversion (only converts 'false' to False in it's own scope)
-
What do You think about that?